lib/redis_memo/memoize_method.rb

```diff
@@ -8,6 +8,28 @@
 require_relative 'util'
 
 module RedisMemo::MemoizeMethod
+
+  # Core entry method for using RedisMemo to cache a method's results. When a method is memoized, all
+  # calls to the method  will first check if the results exist in the RedisMemo cache before calling
+  # the original method.
+  #
+  # @example
```

Comment:
Copy and pasted from the readme as a good example of how to use memoize_method with dependencies